app/react/utils/api.js

```diff
@@ -20,15 +20,58 @@ const doneLoading = data => {
 
 const isNonUsualApiError = error => error.status && ![401, 404, 409, 500].includes(error.status);
 
+const errorMessages = [
+  {
+    key: /PayloadTooLargeError: request entity too large/g,
+    message: 'The request has too large data. Please review any long value property.',
+  },
+  {
+    key: /max_bytes_length_exceeded_exception. Invalid Fields: (.+)/g,
+    message: 'The request has too large data. Please review the follow fields: {0} ',
+  },
+  {
+    key: /ERROR Failed to index documents: (.+)/g,
+    message: 'The data have not saved completely, it failed at indexing: {0} ',
```

Comment:
For singular nouns, you use “has”.  So this should read something like: “The data has not been saved in its entirety” or something like that.

But here, what exactly does “completely” mean? I think we should keep as close as possible to the original error.

“Failed to index documents: {0}”

Why the different message?

Reply:
I was trying to provide more information to the user, but I agree, it was confusing.